helper/src/components/addonsTab.js

```diff
@@ -422,6 +422,22 @@ export default function ({ tabValues, updateFn, featureFlag, invalidArray }) {
                     label="Enable high availability mode"
                 />
             </Stack.Item>
+
+            <Separator className="notopmargin" />
+
+            <Stack.Item align="start">
+                <Label required={true}>
+                    Azure Blob storage CSI (Container Storage Interface - preview): Enable blob access from the cluster using BlobFuse or Network File System (NFS) version 3.0 protocol
+                    (<a target="_new" href="https://docs.microsoft.com/en-gb/azure/aks/azure-blob-csi">docs</a>)
```

Comment:
Can you take out "en-gb/" - we're trying to use non regional page references

Reply:
done!